test_util/helpers.py

```diff
@@ -31,6 +30,35 @@ def authenticate(self, cluster):
         logging.info('Authentication successful')
 
 
+class ApiClient:
+    def __init__(self, cluster, user, api_base, default_headers={}):
+        self.cluster = cluster
+        self.user = user
+        self.api_base = api_base
+        self.headers = default_headers
+
+    def api_request(self, method, path, **kwargs):
+        """Wrapper users the set user to this API instance's user,
+        provides required headers, appends api base, and passes all else
+        """
+        cluster_request = getattr(self.cluster, method)
```

Comment:
this doesn't seem like getattr is really needed. Why can't this just do a little bit of wrapping and call `self.cluster.cluster_request(method, ...)`?